pkg/types/baremetal/validation/platform.go

```diff
@@ -358,9 +366,11 @@ func ValidateProvisioning(p *baremetal.Platform, n *types.Networking, fldPath *f
 			}
 		}
 
-		// Ensure clusterProvisioningIP is in one of the machine networks
-		if err := validateIPinMachineCIDR(p.ClusterProvisioningIP, n); err != nil {
-			allErrs = append(allErrs, field.Invalid(fldPath.Child("clusterProvisioningIP"), p.ClusterProvisioningIP, fmt.Sprintf("provisioning network is disabled, %s", err.Error())))
+		// If set, ensure clusterProvisioningIP is in one of the machine networks
+		if p.ClusterProvisioningIP != "" {
+			if err := validateIPinMachineCIDR(p.ClusterProvisioningIP, n); err != nil {
+				allErrs = append(allErrs, field.Invalid(fldPath.Child("clusterProvisioningIP"), p.ClusterProvisioningIP, fmt.Sprintf("provisioning network is disabled, %s", err.Error())))
```

Comment:
Understanding that this is existing code, this still seems like a good time to fix this.
```suggestion
				allErrs = append(allErrs, field.Invalid(fldPath.Child("provisioningHostIP"), p.ClusterProvisioningIP, fmt.Sprintf("provisioning network is disabled, %s", err.Error())))
```

Reply:
Or I suppose it will conflict with the changes made in https://github.com/openshift/installer/pull/4053, and that other PR is a better place to make the field name consistent.

Reply:
Yea good point, I think when one lands the other will have a conflict so I'll adjust the one that doesn't merge.